<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_LinePolygonOptions</class>
 <widget class="QWidget" name="Ui_LinePolygonOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>24</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Polygon Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="lNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="cbRadius">
     <property name="toolTip">
      <string>If checked, vertexes of polygon will be rounded. </string>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="cbPolyline">
     <property name="toolTip">
      <string>If checked, the polygon will be drawn as polyline. Otherwise, individual entities will be created. </string>
     </property>
     <property name="text">
      <string>Polyline</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbNumber">
     <property name="toolTip">
      <string>Number of edges</string>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="leRadius">
     <property name="toolTip">
      <string>Rounding radius for vertex bevel.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QCheckBox" name="cbVertexToVertex">
     <property name="toolTip">
      <string>If selected, reference points are for vertexes of polygon. Otherwise, they are for middle point of edges. </string>
     </property>
     <property name="text">
      <string>Vertex to Vertex</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>updateNumber(int)</slot>
 </slots>
</ui>
